xml_inscriptions/transcribed/NJ.Princ.PU.AM.G.4740-I130a.xml

```diff
@@ -172,16 +175,61 @@ changes: Added @evidence to date
+                <lb n="3"/>αστρακ ιαηου σαθνανω ωπαχ
+                <lb n="4"/>θαλσου σανταρμα χουχι
+                <lb n="5"/>ερηβαθ σαρδοασιρως αχωρ<add place="above">ω</add>
+                <lb n="6"/>ὦ βρώ<supplied reason="omitted">ν</supplied>των καὶ ἀστ<supplied reason="omitted">ρ</supplied>άπτων <name type="divine" key="Iao Ιαω">Ιαω</name>
+                <lb n="7"/>βάλη δῆσων σύ<supplied reason="omitted">ν</supplied>δησω<supplied reason="omitted">ν</supplied> <name key="Βαβύλας Babylas">Βαβυλᾶν</name>
+                <lb n="8"/>τὼν λαχανωπώλην ὥ<supplied reason="omitted">ν</supplied> ἤτηκεν
+                <lb n="9"/>εἱ μιηρὰ μέτρα <name key="Διωνυσία Dionysia">Διωνυσία</name> εἱ κὲ
+                <lb n="10"/><name key="Εἱσυχία Hesykhia">Εἱσυχία</name> ὐκῶ<supplied reason="omitted">ντα</supplied> ἐν γιτωνία Μυγδ
+                <lb n="11" break="no"/>ωνιτῶν: ὡς ἔβαλες τὼ ἅ<supplied reason="omitted">ρ</supplied>μα το
+                <lb n="12" break="no"/>ῦ Φαραῶνος, οὕτος βάλη τὶν δύσ
+                <lb n="13" break="no"/><surplus>ω</surplus>ληψιν αὐτοῦ: ὦ βρ<supplied reason="omitted">ό</supplied>ντων καὶ
+                <lb n="14"/>ἀσ<surplus>σ</surplus>τρ<supplied reason="omitted">ά</supplied>πτων <name type="divine" key="Iao Ιαω">Ιαω</name>, ὡς ἐξέκωψ
+                <lb n="15" break="no"/>ης τὰ πρωτότυκα τῖς Ἠγύπ
+                <lb n="16" break="no"/>του, ἄ<supplied reason="omitted">κ</supplied>κωψων τ<unclear>ἄλο</unclear><supplied reason="lost">γα α</supplied> 
+                <lb n="17" break="no"/>ὐτο<supplied reason="omitted">ῦ</supplied> τόσυν <surplus>ΟΣΤ</surplus> ὅσ<surplus>ω</surplus>περ Η<unclear> </unclear><gap reason="lost" quantity="3" unit="character"/>  
+                <lb n="18"/>Σ ἐν ἄρτι καὶ δῖσων κατάδ<unclear>ι</unclear>σ
+                <lb n="19" break="no"/>ων σύνδισων κατάκλιν<supplied reason="omitted">ον</supplied> στρέ
+                <lb n="20" break="no"/>ψατη κλαστήτωσαν μὶ δυ
+                <lb n="21" break="no"/>νιθούσιν κινῖ<supplied reason="omitted">σ</supplied>θ<supplied reason="omitted">αι</supplied> τὰ αὐτοῦ <name key="Βαβύλας Babylas">Βα
+                <lb n="22" break="no"/>βυλᾶ</name> τὰ ἄλωγα αὐτοῦ τόσ
+                <lb n="23" break="no"/>ων ὅα<supplied reason="omitted">ον</supplied> αὐποὺ τῖς ὥρας ταύτις
+                <lb n="24"/>καὶ τῖς ἑρήμας ταύτις ἤδη
+                <lb n="25"/>ἤδη ταχὺ ταχὺ κακιμερίας
+                <lb n="26"/>δυσιμερίας π<supplied reason="omitted">λ</supplied>ήσατη τοῦ αὐτοῦ
+                <lb n="27"/><name key="Βαβύλας Babylas">Βαβυλᾶ</name> λαχανωπώλι ὥν ἤτηκ
+                <lb n="28" break="no"/>εν <name key="Εἱσυχία Hesykhia">Εἱσυχία</name>.
+            </ab></div>
             
             <div type="translation">
-            <p></p></div>
+            <p>
+                <hi rend="italic">Britakulma arôdasidaki[.]a 
+                    (2) êllim abri makhrêmella astrak iaêou sathnanô ôpakh 
```

Comment:
We'll have to see how this renders once committed to main; I think if we want to keep the lines as they are here we might need to put them in their own `<p>` elements, but we can revisit that after we see how it looks.